sections/semantics-embedded-content.include

```diff
@@ -898,13 +899,13 @@
   settings attribute</a>. Its purpose is to allow images from third-party sites that allow
   cross-origin access to be used with <{canvas}>.
 
-  The <dfn element-attr for="img"><code>referrerpolicy</code></dfn> attribute is a <a>referrer policy attribute</a>. Its purpose is to
-  set the <a>referrer policy</a> used when <a>fetching</a> the image. [[!REFERRERPOLICY]]
+  The <dfn element-attr for="img"><code>decoding</code></dfn> attribute is an enumerated attribute.
+  It is an  [=image decoding hint=], to reqeust synchronous asynchronous image loading.
```

Comment:
I think there’s an "or" missing between "synchronous asynchronous".
(And does the comma have a function in the same sentence?)

Reply:
"or": yep. +typo in "request", and the comma is ok but unnecessary.

all fixed - thank you